### PR TITLE
small updates to press page

### PR DESCRIPTION
**Pull request by altsalt**
@funnelfiasco made some small changes to the text, mostly new links. Still need to actually add a JPG file. @AndrewKvalheim could you perhaps do that?

### CODE REVIEW

**Review by funnelfiasco — APPROVED** [review]
Shipit

**Comment by AndrewKvalheim** [comment]
The logo should never exist as a JPEG, and if someone is requesting it in that format I’m inclined to gently push back by making them do the work and damage.

File: press.html [hunk 1]
```diff
@@ -11,27 +11,31 @@
     <div class="col-md-8">
       <h2>About SeaGL</h2>
       <p>
-      SeaGL (<em>the Seattle GNU/Linux conference</em>) is a free—as in freedom and tea—grassroots technical summit dedicated to spreading awareness and knowledge about free / libre / open source software, hardware, and culture.
+      SeaGL (the <em>Seattle GNU/Linux conference</em>) is a free—as in freedom and tea—grassroots technical summit dedicated to spreading awareness and knowledge about free / libre / open source software, hardware, and culture.
```
Comment (AndrewKvalheim) [inline press.html:14]: While were here, technically this should use [`<i>`](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-i-element) not emphasis.

Comment (altsalt) [reply press.html:14]: huh, interesting, I didn't realize `<em>` and `<strong>` had been superseded. Anyhoo, not going to update it atm, but if any more fiddling is done, fine with that change. BTW, thoughts on where the emphasis goes? I'm thinking just around the Seattle GNU/Linux bit.

Comment (AndrewKvalheim) [reply press.html:14]: My recommendation is no [emphasis](https://html.spec.whatwg.org/multipage/text-level-semantics.html#the-em-element) at all. Use `<i>` around the expanded name of the conference, once when introducing it, to indicate that it’s “in an alternate voice or […] offset from the normal prose in a manner indicating a different quality of text, such as […] a technical term”. 

As to what the expanded name of the conference is, I think most precedent supports “Seattle GNU/Linux Conference” but perhaps we need to decide that more explicitly.

Comment (AndrewKvalheim) [reply press.html:14]: Straying a bit, I would use emphasis here though:

> free—as in freedom <em>and</em> tea

Comment (altsalt) [reply press.html:14]: all of these points are good, I suppose I can just update them directly to close this out. Unsure about the upper or lowercase C for conference. This whole emphasis line of thinking came from the way that the description on archive.org was changed, in a way that I generally approved of.

**Pull request merged**